tests/frontend/systolic/array-2.expect

```diff
@@ -4,14 +4,14 @@ import "primitives/pipelined.futil";
 static<1> component mac_pe(top: 32, left: 32, mul_ready: 1) -> (out: 32) {
   cells {
```

Comment:
These larger files don't seem to have terribly utility anymore IMO because the changelogs are two big. Maybe we can just have one golden test for this and remove the others?